docs/pages/jupedsim/jupedsim_install_on_windows.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Install JuPedSim on Windows
+tags: [getting_started, troubleshooting]
+summary: "Windows executables will be released. This page gives instructions how to compile the code on windows using Visual Studio"
+keywords: installation
+sidebar: jupedsim_sidebar
+folder: jupedsim
+permalink: jupedsim_install_on_windows.html
+last_updated: Dec 20, 2019
+---
+
+
+## Get the code and create a VS-project
```

Comment:
This page needs to be changed. Either we fix how to build on Windows or we recommend downloading the binaries from the release. The current description is not correct. If all dependencies are installed the best way to build on windows is via Powershell and cmake. Basically it is not much different from Linux except for the installation of dependencies.

Reply:
Do you think updating installation recommendation on windows should be done in this PR already?